packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/upgrading-to-2025-10.doc.ts

```diff
@@ -298,43 +298,44 @@ Use the comparison table below to see which Polaris web components are available
-|   \`POSBlockRow\`            |                                                           |   Coming soon                          |
-|   \`POSReceiptBlock\`        |                                                           |   Coming soon                          |
-|   \`PrintPreview\`           |                                                           |   Coming soon                          |
-|   \`QRCode\`                 |                                                           |   Coming soon                          |
-|   \`RadioButtonList\`        |                                                           |   Coming soon                          |
-|   \`Screen\`                 |                                                           |   Coming soon                          |
-|   \`ScrollView\`             |                                                           |   Coming soon                          |
-|   \`SearchBar\`              |                                                           |   Coming soon                          |
-|   \`Section\`                |                                                           |   Coming soon                          |
-|   \`SectionHeader\`          |                                                           |   Coming soon                          |
-|   \`SegmentedControl\`       |                                                           |   Coming soon                          |
-|   \`Selectable\`             |                                                           |   Coming soon                          |
-|   \`Spacing\`                |                                                           |   Coming soon                          |
-|   \`Stack\`                  |                                                           |   Coming soon                          |
-|   \`Stepper\`                |                                                           |   Coming soon                          |
-|   \`Text\`                   |                                                           |   Coming soon                          |
-|   \`TextArea\`               |                                                           |   Coming soon                          |
-|   \`TextField\`              |                                                           |   Coming soon                          |
-|   \`Tile\`                   |                                                           |   Coming soon                          |
-|   \`TimeField\`              |                                                           |   Coming soon                          |
-|   \`TimePicker\`             |                                                           |   Coming soon                          |
+|   \`Badge\`                  |   [\`Badge\`](/docs/api/pos-ui-extensions/2025-10-rc/polaris-web-components/titlesandtext/badge)   |   Available   |
```

Comment:
I believe these links will work (and will keep the user in the same version of the docs) if you just do:

```suggestion
|   \`Badge\`                  |   [\`Badge\`](/polaris-web-components/titlesandtext/badge)   |   Available   |
```

Reply:
Good catch, as long as you don't have a leading `/` it'll work as `polaris-web-components/titles-and-text/badge`